test/integration_helper.rb

```diff
@@ -77,6 +77,12 @@ def go_to_dashboard
   end
 
   def click_away_from_field
+    # close the flash if it's in the way
+    begin
+      find('#flash button.close')&.click
```

Comment:
is this ok? (versus *first* check if the element, if it does click it)

Reply:
I think this is a great approach!